public/lesswrong-editor/src/spoilers-plugin.ts

```diff
@@ -5,6 +5,17 @@ import Widget from '@ckeditor/ckeditor5-widget/src/widget';
 import Command from '@ckeditor/ckeditor5-core/src/command';
 import blockAutoformatEditing from '@ckeditor/ckeditor5-autoformat/src/blockautoformatediting';
 import first from '@ckeditor/ckeditor5-utils/src/first';
+import type { Element } from '@ckeditor/ckeditor5-engine';
+import type Writer from '@ckeditor/ckeditor5-engine/src/model/writer';
+import type Schema from '@ckeditor/ckeditor5-engine/src/model/schema';
```

Comment:
Typescript still seems to be confused about where it is for me. It's possible this is a vscode issue rather than an issue with the tsconfig, @s-cheng do you get this too?

![Screenshot 2024-06-24 at 12 16 37](https://github.com/ForumMagnum/ForumMagnum/assets/77623106/087e9b4f-d863-4f66-8074-51de4796b6c4)

Reply:
Maybe this corresponds to needing to run `yarn install` in `public/lesswrong-editor`?

Reply:
This turned out to be caused by having `jsconfig.json` present in the project, and fixed by removing them